### ISSUE TITLE
FastTracker II module sounds strange

### ISSUE THREAD

**Issue by neumatho** [libxmp/libxmp#683]
Hi,

I have found a module, which sounds very strange. It happens at position 3, channel 17-21. When looking in OpenMPT (which plays it fine), it seems that the channels uses the Z effect (MIDI) with different values. Maybe some specific implementation is missing?

[Fairy Tales Remix.zip](https://github.com/libxmp/libxmp/files/12708254/Fairy.Tales.Remix.zip)

**Comment by sezero** [comment]
@AliceLR ?

**Comment by sagamusix** [comment]
I don't think libxmp currently supports Zxx in XMs (which is a ModPlug extension). It could be modified to do so, but should only do that when the file appears to be saved with (Open)MPT, to not break XMs made with FT2 where the author accidentally (or deliberately) entered a Z into the effect column.

**Comment by AliceLR** [comment]
Yes, I'd like to add support for this at some point now that there's (slightly) better MPT detection. MIDI macros are currently isolated to IT because I added them before improved XM MPT detection was required.
